Sources/xcproj/Deprecations.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Don't forget to remove the headers @rahul-malik! I'll add a Danger task to chek that automatically.